src/sttn/eval/eval_runner.py

```diff
@@ -0,0 +1,159 @@
+import nbformat

```

Comment:
Does your notebook consume this script? We probably don't need to check it in now if it's not working.

Reply:
Yes, the runner script needs this library to run the notebook